Add Get All Saved Addresses endpoint  #9

Khilan10 opened the pull request:
- Add mapping /address/customer to get all saved address in descending order of time

- Added name queries in Address Entity and Customer Address Entity

Khilan10 committed:
Add Get All Saved Addresses endpoint  #9

- Add mapping /address/customer to get all saved address in descending order of time

- Added name queries in Address Entity and Customer Address Entity

pavankarthiktanikanti reviewed (changes requested):
- Use  One named query in CustomerAddressEntity to fetch the entities

- Prefer to use Id attributes of Entities when comparing, instead of passing object directly

- Extract the AddressEntity from each CustomerAddressEntity object

- Follow NamedQuery Declaration Syntax

- Remove unnecessary NamedQuery in AddressEntity

Khilan10 committed:
Updating named Query for get All Address Endpoint #9

- Removed Named Query in Address Entity and changed Named Query in Customer Address Entity

- Changed Address Dao according to the named query

Khilan10 committed:
Updated named query 'allAddressForCustomer" to sort  Address Entity  in descending order of  id  #9

pavankarthiktanikanti reviewed (changes requested):
- Modify order by clause to use ca.address.id instead of ca.id in CustomerAddressEntity

Khilan10 committed:
- Update the named query for allAddressForCustomer to do sorting by address id #9

pavankarthiktanikanti commented on FoodOrderingApp-service/src/main/java/com/upgrad/FoodOrderingApp/service/entity/CustomerAddressEntity.java:
```diff
@@ -9,6 +9,10 @@
 
 @Entity
 @Table(name = "CUSTOMER_ADDRESS")
+@NamedQuery(name = "allAddressesForCustomer", query = "select ca.address from CustomerAddressEntity ca where ca.customer.id = :customerId order by ca.id desc")
```
- Remove Duplicate Named Query declaration

pavankarthiktanikanti reviewed (changes requested):
- Remove Duplicate NamedQuery declaration

Khilan10 committed:
- Removing duplicate named query #9

pavankarthiktanikanti reviewed (approved):
- reviewed changes for get all customer address
